### PR TITLE
PR#68: Implemented a search bar for user page. 

### PR DESCRIPTION
**Pull request by kpcasta**
I updated the html for the user page to include a search bar. I am having some troubles with css styling, however this can be fixed later on. In the coming week I will apply java/js to give the search bar functionality. 
Screen Shot 2019-03-31 at 9 30 17 AM

### CODE REVIEW

**Comment by FluffySheep** [comment]
Seems like you are pushing to the wrong branch. Please change to develop.

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+    if (!userService.isUserLoggedIn()) {
+      response.sendRedirect("/index.html");
+      return;
+    }
+
+    String aboutMe = Jsoup.clean(request.getParameter("about-me"), Whitelist.none());
```
Comment (linkaren) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: In my own previous pull requests, I had changed this Whitelist.none() to Whitelist.basicWithImages() along with some other changes. Is there no merge conflict happening here? @FluffySheep

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 2]
```diff
@@ -77,10 +77,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     String user = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
```
Comment (linkaren) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java]: Same thing here, I'm not really sure why my previous changes aren't showing up.

**Comment by linkaren** [comment]
Oh, you're comparing with master. My comments should be resolved when comparing to develop!

**Comment by FluffySheep** [comment]
Please perform:

```
git reset --soft 7707cd18ffb1a06c3218aa71810979179fed690f
git rebase develop
```

**Pull request closed without merging**